### PR TITLE
LAD: Reinstall OMI if it is failing to start as recovery action

### PR DESCRIPTION
**Pull request by canfikret**
We got a few CRIs where OMI service does not start due to a possible corruption. LAD detects that OMI is not available; it attempts to restart the service. If still a problem, the component is re-installed.

### CODE REVIEW

File: Diagnostic/diagnostic.py [hunk 1]
```diff
@@ -311,7 +311,7 @@ def main(command):
                 dependencies_err, dependencies_msg = setup_dependencies_and_mdsd(configurator)
                 if dependencies_err != 0:
                     g_lad_log_helper.report_mdsd_dependency_setup_failure(waagent_ext_event_type, dependencies_msg)
-                    hutil.do_status_report(g_ext_op_type, "error", '-1', "Enablecd  failed")
+                    hutil.do_status_report(g_ext_op_type, "error", '-1', "Enabled failed")
```
Comment (AaronRM) [inline Diagnostic/diagnostic.py]: Enabled=>Enable?

File: Diagnostic/diagnostic.py [hunk 2]
```diff
@@ -644,7 +644,21 @@ def restart_omi_if_crashed(omi_installed, mdsd):
                         "Restarting OMI and sending SIGHUP to mdsd after 5 seconds.")
```
Comment (AaronRM) [inline Diagnostic/diagnostic.py]: Change message to say 10 seconds.

File: Diagnostic/diagnostic.py (hunk 1 above)
Comment (canfikret) [reply Diagnostic/diagnostic.py]: yep :-)

File: Diagnostic/diagnostic.py (hunk 2 above)
Comment (canfikret) [reply Diagnostic/diagnostic.py]: right.

**Pull request merged**